lms/envs/common.py

```diff
@@ -1269,19 +1269,17 @@
         ],
         'output_filename': 'css/lms-style-vendor-tinymce-skin.css',
     },
-    'style-app': {
+    'style-main': {
         'source_filenames': [
-            'sass/application.css',
-            'sass/ie.css'
+            'sass/lms-main.css',
         ],
-        'output_filename': 'css/lms-style-app.css',
+        'output_filename': 'css/lms-main.css',
```

Comment:
@feanil, will this have any cache/production repercussions when this gets merged and a release is cut?